app/models/article.rb

```diff
@@ -28,6 +28,8 @@ class Article < ApplicationRecord
   counter_culture :organization
 
   has_many :comments, as: :commentable, inverse_of: :commentable
+  has_many :user_subscriptions, as: :user_subscription_sourceable
+  has_many :subscribers, class_name: "User", through: :user_subscriptions, foreign_key: :user_id
```

Comment:
I think we should take this out completely or rename it to something along the lines of `sourced_author_subscribers` or `sourced_subscribers` . The reason I suggest this is bc the users are not subscribing to the article they are subscribing to the author and I think this relationship could make that confusing.

Reply:
I'm game, how about `source_author_subscribers` (`source` > `sourced`)?

Reply:
Updated to `sourced_subscribers` 🚀